pkg/api/v1alpha1/cluster_webhook.go

```diff
@@ -58,14 +58,16 @@ func (r *Cluster) ValidateCreate() error {
 		clusterlog.Info("cluster is paused, so allowing create", "name", r.Name)
 		return nil
```

Comment:
this will skip the validation, I don't think we want that

Maybe we need to reorder all these statement in validate create

Reply:
even with the pause annotation we want the validations to be checked for?
I can move the logic around then.

Reply:
I would vote to always validate, even with annotation, yes